lib/Runtime/Language/InterpreterLoop.inl

```diff
@@ -157,14 +157,25 @@ SWAP_BP_FOR_OPCODE:
                 return nullptr;
 
             case INTERPRETER_OPCODE::ExtendedOpcodePrefix:
+            case INTERPRETER_OPCODE::DblExtendedOpcodePrefix:
```

Comment:
But if DblExtendedOpcodePrefix gets its own case arm per your suggestion above, then we will have to replicate the code below, and that case is never taken on non-asm.js interpreter.